src/sacn/private/opts.h

```diff
@@ -253,10 +258,10 @@
 /** @endcond */
 
 /**
- * @brief Currently unused; will be used in the future.
+ * @brief Currently unused; will be used in the future. Keep this set to 1.
```

Comment:
Instead of telling the user to keep it set to 1, we should just do it:
```
#undef SACN_RECEIVER_MAX_THREADS
#define SACN_RECEIVER_MAX_THREADS 1
```